### PR TITLE
Stop using the beta.kubernetes.io/arch and os

### PR DESCRIPTION
**Pull request by pacoxu**
See https://github.com/kubevirt/kubevirt/pull/8684

### CODE REVIEW

**Comment by brianmcarey** [comment]
@pacoxu  thanks for the PR - could you please sign your commit ? (https://github.com/kubevirt/kubevirt/blob/main/CONTRIBUTING.md#contributor-compliance-with-developer-certificate-of-origin-dco)

**Comment by pacoxu** [comment]
> @pacoxu thanks for the PR - could you please sign your commit ? (https://github.com/kubevirt/kubevirt/blob/main/CONTRIBUTING.md#contributor-compliance-with-developer-certificate-of-origin-dco)

Done. Thanks for reminding.

**Comment by pacoxu** [comment]
All tests are passed.

**Comment by pacoxu** [comment]
/assign @phoracek
for approval

File: cluster-provision/k8s/1.25/manifests/whereabouts/whereabouts-v0.5.4.yaml [hunk 1]
```diff
@@ -284,7 +284,7 @@ spec:
           name: cni-net-dir
       hostNetwork: true
       nodeSelector:
-        beta.kubernetes.io/arch: amd64
+        kubernetes.io/arch: amd64
```
[the remark this answers is not in the document]
Comment (pacoxu) [reply cluster-provision/k8s/1.25/manifests/whereabouts/whereabouts-v0.5.4.yaml:287]: I opened https://github.com/openshift/whereabouts-cni/pull/106 for whereabouts.

[the remark this answers is not in the document]
Comment (maiqueb) [reply cluster-provision/k8s/1.25/manifests/whereabouts/whereabouts-v0.5.4.yaml:287]: Merged https://github.com/k8snetworkplumbingwg/whereabouts/pull/287; lgtm'ed the other one.

**Pull request closed without merging**